cmd/root.go

```diff
@@ -103,6 +103,10 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVarP(&lagoonYml, "lagoon-yml", "l", ".lagoon.yml",
 		"The .lagoon.yml file to read")
+	rootCmd.PersistentFlags().StringVarP(&lagoonYmlOverride, "lagoon-yml-override", "", ".lagoon.override.yml",
+		"The .lagoon.yml override file to read for merging values into target lagoon.yml")
+	rootCmd.PersistentFlags().StringVarP(&lagoonYmlEnvVar, "lagoon-yml-envvar", "", "LAGOON_YAML_OVERRIDE",
```

Comment:
Do you think this would be something people would want to change?

Reply:
@shreddedbacon - yeah, I think it's _possible_ - happy to move it into the validate command only if you want (which is where I think it'll mostly be used, if at all).
I just wired it up like this to make it a little more flexible.

Reply:
I'm just struggling to find why someone would want to use a different named envvar to populate?

Reply:
@shreddedbacon - I guess it's really less about the name than I wondered about making it fixed. I'll pull it out.
I think the override file name is probably useful though.

Reply:
Yeah, I'm ok with filename override. If someone was doing something locally they're more likely to be using files than base64 encoding variables, so it makes more sense to keep the filename flag. But if they're going to the point of base64 encoding values, I think supporting just 1 envvar is perfectly fine.

Reply:
@shreddedbacon done :)